armi/reactor/converters/axialExpansionChanger.py

```diff
@@ -175,9 +192,10 @@ def axiallyExpandAssembly(self, thermal: bool = False):
             # if ib == 0, leave block bottom = 0.0
             if ib > 0:
                 b.p.zbottom = self.linked.linkedBlocks[b][0].p.ztop
-            # if not in the dummy block, get expansion factor, do alignment, and modify block
-            if ib < (numOfBlocks - 1):
+            isDummyBlock = ib == (numOfBlocks - 1)
```

Comment:
Can this be based on flags?

Reply:
scope creep.... they are treated as flags in other places, but this is a potential separate issue.

there are warnings if they are not dummy
![image](https://user-images.githubusercontent.com/50242044/185217711-2e8f6e1f-5351-4e65-9e50-4a1b8b38601b.png)

Reply:
@jakehader - I don't think it should be, actually. If we were to change it to be something like 

```python
if not b.hasFlags(Flags.DUMMY):
    # do the expansion/contraction on the block
``` 

then we'd be expanding/contracting the top-most block of an assembly if it is not have a dummy  block. This could be problematic for downstream plugins as some require the assemblies to all be the same height. 

By leaving it as is, we ensure that the top-most block, dummy or not, is chopped to preserve assembly height.

Reply:
Got it! Thank you for the clarification. I'm good then, but note I haven't done a complete review of the work or tests so I am leaving that to @onufer or other reviewers unless I'm needed.